CONTRIBUTING.md

```diff
@@ -18,6 +18,15 @@ There are many ways to contribute to Docusaurus, and many of them do not involve
 
 Contributions are very welcome. If you think you need help planning your contribution, please ping us on Twitter at [@docusaurus](https://twitter.com/docusaurus) and let us know you are looking for a bit of help.
 
+You might want to make yourself familiar with [docusaurus](https://docusaurus.io/docs/en/installation) to make significant changes to the website. If you only want to make content changes you just need to know about versioned docs.
```

Comment:
This is not relevant because for our entire project (website + library), the users have to be familiar with Docusaurus. I'll change this.